test/claims-status/actions.unit.spec.js

```diff
@@ -205,7 +205,7 @@ describe('Actions', () => {
     it('should fetch claims', (done) => {
       const appeals = [];
       fetchMock.returns({
-        then: (fn) => fn({ ok: true, json: () => Promise.resolve(appeals) })
+        'catch': () => ({ then: (fn) => fn({ ok: true, json: () => Promise.resolve(appeals) }) }),
```

Comment:
Does eslint not complain about unnecessary `'`s?

Reply:
It in fact complains if I don't use them, because `catch` is a reserved word.